### ISSUE TITLE
"Condition ' !is_inside_tree() ' is true. returned: NodePath()" during debugging.

### ISSUE THREAD

**Issue by hilfazer** [godotengine/godot#21378]
**Godot version:**

3.0.6

**OS/device including version:**

Windows 7 Pro SP1

**Issue description:**

Debugger started spamming this error:
```
0:00:06:0965 - Condition ' !is_inside_tree() ' is true. returned: NodePath()
----------
Type:Error
Description: 
Time: 0:00:06:0965
C Error: Condition ' !is_inside_tree() ' is true. returned: NodePath()
C Source: scene/main/node.cpp:1780
C Function: get_path
```
I expected no errors. Bugs can be everywhere so i need an ability to inspect nodes that are outside of tree as well. 

**Steps to reproduce:**
Place a breakpoint at 'pass', run the project and click the 'node' variable in Debugger (it's going to be something like 'Object: 967').

I suspected Inspector but this problem appears regardless of Inspector tab being active so i dunno.

**Minimal reproduction project:**

https://github.com/hilfazer/Projects/tree/EngineIssues/Godot/DebuggerSpamsIsInsideTree

**Comment by DualMatrix** [comment]
Could not reproduce in master

**Comment by akien-mga** [comment]
Indeed it seems fixed in the master branch. I could reproduce it in the 3.0 branch though.

**Comment by akien-mga** [comment]
Given that it's not a very critical issue and only happens with nodes not in the tree, I'll close as fixed in 3.1. If someone wants to provide a fix for 3.0.x though, that's welcome.

**Comment by hilfazer** [comment]
I still have this problem in Godot_v3.1-alpha4_win64.
